src/client/store/assessment/slice.ts

```diff
@@ -37,6 +37,10 @@ export const assessmentSlice = createSlice({
       state.countries[payload.countryIso] = payload
     })
 
+    builder.addCase(updateCountryProp.fulfilled, (state, { payload }) => {
```

Comment:
I would use optimistic ui update , which means updating ui on pending action , not fulfilled.
Would that work ?

Reply:
it would work:

`    builder.addCase(updateCountryProp.pending, (state, reducer) => {
      const {
        meta: { arg },
      } = reducer

      const { countryIso, countryProp } = arg

      state.countries[countryIso].props = { ...state.countries[countryIso].props, ...countryProp }
    })`